shanoir-ng-studies/src/test/java/org/shanoir/ng/study/service/StudyServiceTest.java

```diff
@@ -209,11 +285,12 @@ private Study createStudy() {
 		return study;
 	}
 	
-	private StudyUser createStudyUsers(Long suId, Long userId, Study study, StudyUserRight... rights) {
+	private StudyUser createStudyUsers(Long suId, Long userId, Study study, boolean confirmed, StudyUserRight... rights) {
 		StudyUser studyUser = new StudyUser();
 		studyUser.setId(suId);
 		studyUser.setStudy(study);
 		studyUser.setUserId(userId);
+		studyUser.setConfirmed(true);
```

Comment:
Hi Jean-Come, thank you a lot for this great fix! I guess here you want to use the value of confirmed, and not a fix value?

Reply:
I am wondering, how the UnitTest runs in case of that value per default

Reply:
Damn.